pyproject.toml

```diff
@@ -12,3 +12,7 @@ combine_as_imports = true
 [tool.mypy]
 ignore_missing_imports = true
 show_error_codes = true
+mypy_path = "$MYPY_CONFIG_FILE_DIR/stubs"
+
+[tool.pyright]
```

Comment:
This will make VSCode nicely pick up the stubs without further configuration. Maybe we want to go all-in on VSCode as a preferred tool for developers, but that's a bigger commitment and separate discussion.